### DISCUSSION TITLE
Palo Alto firewall as exporter

### DISCUSSION THREAD

**Discussion by capitangiaco** [akvorado/akvorado#720] [Q&A]
Is there anyone who managed to use Palo Alto Networks firewalls as exporter ?

I see only snmp errors:
{"level":"error","error":"request timeout (after 1 retries)","exporter":"192.168.231.251","time":"2023-05-21T10:19:58Z","caller":"akvorado/inlet/snmp/poller.go:186","module":"akvorado/inlet/snmp","message":"unable to GET (7 OIDs)"}
{"level":"error","error":"request timeout (after 1 retries)","exporter":"192.168.231.251","time":"2023-05-21T10:19:58Z","caller":"akvorado/inlet/snmp/poller.go:186","module":"akvorado/inlet/snmp","message":"unable to GET (4 OIDs)"}
{"level":"error","error":"request timeout (after 1 retries)","exporter":"192.168.231.251","time":"2023-05-21T10:19:58Z","caller":"akvorado/inlet/snmp/poller.go:186","module":"akvorado/inlet/snmp","message":"unable to GET (4 OIDs)"}

thanks

**Comment by vincentbernat** [comment]
Is SNMP configured to accept requests on 192.168.231.252?

**Reply by capitangiaco** [reply]
yes, that IP already respond to LibreNMS snmp queries.
The only difference showed in the device logs, is that LibreNMS use snmpv2 while akvorado seems to use v1.

**Reply by vincentbernat** [reply]
I don't think this is possible for an agent to detect if it is being queried with SNMPv1 or SNMPv2. Maybe that's because LibreNMS is using GETBULK while Akvorado uses GET. If the communities are correct, likely the indexes do not exist. Capture with tcpdump the SNMP requests and check manually if they succeed.
